modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -100,3 +136,15 @@ The makefile of the [Stochastic Tools Module](stochastic_tools/stochastic_tools.
 serves as a good example on how to achieve this for applications.
 
 !alert-end!
+
+## GPU Support
+
+When using ARM Macs the conda-based installation supports both CPU and GPU devices.
+For GPU acceleration through Metal, users need to select MPS as a device when writing source code.
+
+!alert! warning
+
+GPU devices on Linux machines are not officially supported yet. If you have questions regarding
+this path, feel free to ask them on out Discussion forum!
+
+!alert-end!
```

Comment:
You said earlier:

> For notes on the GPU support, navigate to the bottom of this site.

But the user will be presented with this message when they actually scroll down here.